spark/src/main/scala/geotrellis/spark/reproject/TileRDDReproject.scala

```diff
@@ -81,6 +82,76 @@ object TileRDDReproject {
           }
       }
 
+    val (sampleKey, BufferedTile(sampleTile, sampleGridBounds)) = bufferedTiles.first() // XXX
```

Comment:
Need `sampleKey` because it is not possible (as far as I know) to construct an object of type K, and I need such an object in order to be sample to build the list of keys (see line 94).

Need `sampleTile` because in the unit tests, the metadata reports that tiles are 50x50, but the actual tiles are of variable size and always larger than 50x50.  The fixed tile size taken from the first tile works for the unit tests, but I do not know if this is a universally viable strategy.  A "good enough" tile size is needed in order for the cellSize and tile-contents checks in the unit tests to pass.

Need `sampleGridBounds` for much the same reason as `sampleTile`.